src/media/js/views/search.js

```diff
@@ -208,12 +208,18 @@ define('views/search',
             if (query) break;
         }
 
-        if (query && queryParam === 'author') {
-            pageTypes += ' leaf';
-            utilsLocal.headerTitle(gettext('Developer Listing'));
-        } else if (!settings.meowEnabled) {
-            pageTypes += ' leaf';
-        }
+        if (query) switch (queryParam) {
```

Comment:
let's put the switch on its own line so it will be wrapped with `if (query) {....}`